README.md

```diff
@@ -28,11 +28,25 @@ For details, see [Luigi documentation](docs/README.md).
 
 All projects in the repository use [Prettier](https://prettier.io) to format source code. Run the `npm install` command in the root folder to install it along with [husky](https://github.com/typicode/husky), the Git hooks manager. Both tools ensure proper codebase formatting before committing it.
 
-### Tests
+### Unit Tests
+
+To ensure that existing features still work as expected after your changes, run unit tests by running `npm run test` the [core](/core) folder.
+
+### E2E Tests
 
 To ensure that existing features still work as expected after your changes, run UI tests from the [Angular example application](/core/examples/luigi-sample-angular). Before running the tests, start the sample application by using the `npm start` command in the application folder.
 
 When the application is ready:
 
-- Run `npm run e2e:run` in the `core/examples/luigi-sample-angular` folder to start tests in the headless browser.
 - Run `npm run e2e:open` in the `core/examples/luigi-sample-angular` folder to start tests in the interactive mode.
+- Run `npm run e2e:run` in the `core/examples/luigi-sample-angular` folder to start tests in the headless browser.
+
+### Backwards Compatibility Tests
+
+To ensure that applications written for previous versions of Luigi still work after Luigi gets updated via npm. Before running the tests, start bundle Luigi by running `lerna run bundle` in the main repo folder.
+
+Install the json command line parser with `brew install jq` which is required for the script. Can be ommited if you run it with latest on the ci.
```

Comment:
Question: I assume that the json command line parser is required for the script to work.
The second part is somehow unclear to me - can the command be omitted if you run it - it means the script? 
Also `with latest on CI` - doesn't say what :)

Reply:
Correct, the command line parser is required to work for the script. With _latest_ I was referring to something like: If you run `npm run test:compatibility -- --tag latest`, jq is not required.